scripts/updateParcel.js

```diff
@@ -0,0 +1,42 @@
+const glob = require('glob');
+const fs = require('fs');
+
+let tag = process.argv[2] || 'latest';
+let versions = new Map();
+
+async function update(pkg) {
```

Comment:
We should be able to solve this with one or two lines in yarn constraints

though network requests would slow constraints down, why does it need to fetch anything from npmjs?

Reply:
when you're on a nightly and you want to update to the latest nightly it's hard because all of the nightly versions are different per package. So this just fetches the latest nightly dist tag from npm and updates the deps

Reply:
but couldn't we just set `@parcel/core: nightly`? no need to actually get the absolute value, the lock file will hold that for us?

Reply:
yeah but then how do you force yarn to update it? the only way I've found that doesn't result in duplicate packages is to manually update it in every package.json

Reply:
i was thinking https://yarnpkg.com/features/constraints#what-can-we-enforce
but i don't know if we can pass in any arguments to constraints such that we can tell it to enforce nightly vs something else

Reply:
i was thinking https://yarnpkg.com/features/constraints#what-can-we-enforce
but i don't know if we can pass in any arguments to constraints such that we can tell it to enforce nightly vs something else

Reply:
but this is working, so I'm not super fussed about it, I'll resolve this